### PR TITLE
Fixes new desktop file locations

### LINKED ISSUE #145: New icon .desktop location
Some .desktop files are in ```/usr/local/share/applications``` instead of the default ```/usr/share/applications```

### PR DESCRIPTION
**Pull request by bil-elmoussaoui**
Fix #145

### CODE REVIEW

File: fix.sh [hunk 1]
```diff
@@ -140,54 +140,54 @@ while read -r name launcher current new_icon; do
...
-            new_current=$(grep -Gq "Icon=*$" "$local_apps$launcher")
-        else
-            for global_app in "${global_apps[@]}"
-            do
-                if [ -f "$global_app$launcher" ]; then
-                   new_current=$(grep -Gq "Icon=*$" "$local_apps$launcher")
-                fi
-            done
-        fi
+      combined_apps=( "${local_apps[@]}" "${global_apps[@]}" )
```
Comment (Foggalong) [inline fix.sh]: If I'm seeing this right, this isn't the right indentation

Comment (bil-elmoussaoui) [inline fix.sh]: I just saw that too ;)

File: fix.sh [hunk 2]
```diff
@@ -15,10 +15,10 @@ date=201508240  # [year][month][date][extra]
```
Comment (Foggalong) [inline fix.sh]: Make sure to bump the version number (in this case to `201508241`) so that users will be notified of updates

**Comment by bil-elmoussaoui** [comment]
@Foggalong I've added a verifiation to check if the file already exists in hicolor theme, that seems to be the right thing to do rather then overwriting it, what do you think?

**Comment by Foggalong** [comment]
That sounds like a good idea :)

**Comment by bil-elmoussaoui** [comment]
Just one last thing before you merge this, should i use spaces or tabs for indentation?

**Comment by Foggalong** [comment]
Spaces :)

**Comment by bil-elmoussaoui** [comment]
@Foggalong i fixed all the indentation ;)

**Comment by Foggalong** [comment]
Thanks!

**Pull request merged**